docs/site/migration/LB3-vs-LB4-booting.md

```diff
@@ -0,0 +1,153 @@
+[Application](./apidocs/apidocs.core.application.md) class
+and the
+[@loopback/boot](./apidocs/apidocs.boot.html) package. Since your application
+will extend `BootMixin` and `Application`, their separation will not be
+apparent to you, and understanding of how they work together as one is not
+mandatory for developing LoopBack 4 projects.
+
+The following are the list of tasks that are performed in the LoopBack 3
+booting process, and their equivalent in LoopBack 4.
+
+{% include tip.html content="For all the details about booting in LoopBack 4,
+refer to the
+[booting documentation](./Booting-an-Application.md)." %}
+
+### 1. Configuration of application settings
+
+There is no `config.json` in LoopBack 4. The application options are passed in
+the constructor of the
+[Application](./apidocs/apidocs.core.application.md) class. Read more about
+application configuration
+[here](./Application.md#configuring-your-application).
+
+### 2. Configuration of datasources
+
+Datasources in LoopBack 3 are defined in a single `datasources.json` file. In
+LoopBack 4, datasources are defined as TypeScript classes that come with an
+associated JSON configuration file. They all reside in the `src/datasources`
+directory.
+
+{% include note.html content="We are moving away from JSON files for datasource
+configuration. Refer to
+[loopback-next#5000](https://github.com/strongloop/loopback-next/pull/5000)
+for the progress." %}
+
+`BootMixin` resolves the datasources, and attaches them to the application.
+Read more about datasource booter
+[here](./Booting-an-Application.md#controller-booter).
+
+Read more about LoopBack 4 datasource [here](./DataSources.md).
+
+### 3. Definition of models
+
+Although REST APIs are built around Models, they are not a part of the booting
+process in LoopBack 4; Controllers and Repositories are.
+
+Controllers and Repositories import models as regular TypeScript classes since
+they are required at compile time.
+
+{% include note.html content="Booters are LoopBack 4 artifacts that are used
+for modularly adding functionality to the framework. Controllers, Repositories,
+Datasources, Interceptors, etc., are added using booters, custom booters can
+be written for adding new functionality to the framework." %}
+
+#### 4. Setting up middleware
+
+Middleware are not supported at LoopBack level yet. However, you can
+[mount a LoopBack 4 application](./express-with-lb4-rest-tutorial.md) on an
+Express application, which would allow you to use the familiar Express routing
+methods.
+
+{% include tip.html content="Follow our GitHub issues
+[#1293](https://github.com/strongloop/loopback-next/issues/1293)
+and
+[#2035](https://github.com/strongloop/loopback-next/issues/2035)
+to track the progress on supporting Express middlweware in LoopBack 4." %}
+
+If you want to mount an Express router in a LoopBack 4 application, you can use
+the
+[RestApplication.mountExpressRouter()](./apidocs/apidocs.rest.restapplication.mountexpressrouter.md)
```

Comment:
Can we link to https://loopback.io/doc/en/lb4/Routes.html#mounting-an-express-router instead?